### ISSUE TITLE
Hide some of the information from output when activating an environment

### ISSUE THREAD

**Issue by haugstve** [conda/conda#8284]
After updating Conda yesterday I started seeing the following message when I activate environments. (I think it is related but I am not sure).

```
INFO: deactivate-gfortran_osx-64.sh made the following environmental changes:
-DEBUG_FFLAGS=-march=nocona -mtune=core2 -ftree-vectorize -fPIC -fstack-protector -O2 -pipe -march=nocona -mtune=core2 -ftree-vectorize -fPIC -fstack-protector -O2 -pipe -Og -g -Wall -Wextra -fcheck=all -fbacktrace -fimplicit-none -fvar-tracking-assignments
-F95=/anaconda3/bin/x86_64-apple-darwin13.4.0-gfortran
-FC=/anaconda3/bin/x86_64-apple-darwin13.4.0-gfortran
-FFLAGS=-march=nocona -mtune=core2 -ftree-vectorize -fPIC -fstack-protector -O2 -pipe
-FORTRANFLAGS=-march=nocona -mtune=core2 -ftree-vectorize -fPIC -fstack-protector -O2 -pipe
-GFORTRAN=/anaconda3/bin/x86_64-apple-darwin13.4.0-gfortran
```

Today the following message promoted me to update again.

```
 ==> WARNING: A newer version of conda exists. <====
  current version: 4.6.3
 latest version: 4.6.4

Please update conda by running

 $ conda update -n base conda 
```

The message is stil there after the update.

I have never coded anything in FORTRAN so the whole message seems a bit strange. I'm more of a mathematician than a developer, so  how this happened is beyond me. All I know it that it felt a bit overwhelming . For me the message corresponded with some dependencies issues and sendt me on a goose chase. 

I appreciate the great work going into Conda. This feedback is my way of trying to contribute. I know it's not much but i'm not qualified to contribute by making pull requests :)

This is partly related to  #6945 but I have not (to my knowledge) done anything to install or update compilers. I only work in Python and R.

**Comment by msarahan** [comment]
All of the activate and deactivate scripts are here:

Linux: https://github.com/AnacondaRecipes/aggregate/tree/master/ctng-compilers-activation-feedstock/recipe
mac: https://github.com/AnacondaRecipes/aggregate/tree/master/gfortran-feedstock/recipe

as far as I can see, they all have logic to only show the output either when CONDA_BUILD is set, or when an error occurs:

https://github.com/AnacondaRecipes/aggregate/blob/master/gfortran-feedstock/recipe/deactivate-gfortran.sh#L113-L126

You might need to update your gfortran_osx-64 package to make the messages go away.  Older builds might not have that output hiding code.

Even though you don't directly use fortran, one or more of the tools in your environment do use fortran (probably the R stuff, it depends on having compilers present).  That's why you're getting this package and thus seeing these messages.

**Comment by haugstve** [comment]
Thank you so much for the feedback. 

You were quite right. There is something strange going on that seems to be more a local problem. 

In .bash_profile i run the below script. It means I'm always working from base anaconda or from an environment. 
```
. /anaconda3/etc/profile.d/conda.sh                                              
conda activate
```

It seems the problem occurs when I run `conda deactivate` from the base environment.

**Comment by haugstve** [comment]
This has brought me close doing a fresh install of my mac, but I think I figured it out. 

It seems to be the result of running conda activate twice. First time you activate the base environment and the second time I do it, the message pops up. 

This might not seem like a big deal but I'm not the one running that was activating the same environment  twice. It was the terminal in VS code that ran my bash_profile script and started activating a new environment before the script completed.

In my .bash profile I got these two lines.
```
. /anaconda3/etc/profile.d/conda.sh
conda activate # Puts the root environment on path 
```
